api4/system.go

```diff
@@ -23,6 +23,7 @@ func (api *API) InitSystem() {
 	api.BaseRoutes.ApiRoot.Handle("/config", api.ApiSessionRequired(updateConfig)).Methods("PUT")
 	api.BaseRoutes.ApiRoot.Handle("/config/reload", api.ApiSessionRequired(configReload)).Methods("POST")
 	api.BaseRoutes.ApiRoot.Handle("/config/client", api.ApiHandler(getClientConfig)).Methods("GET")
+	api.BaseRoutes.ApiRoot.Handle("/config/environment", api.ApiSessionRequired(getEnvironmentConfig)).Methods("GET")
```

Comment:
Seems excessive to add a new route for this. Could we include it with the getClientConfig somehow?

Reply:
Joram made a similar comment when I submitted the API documentation, but neither of us had any good ideas for how to do that. I'll post this in the Developer channel on pre-release to see if anyone has any good ideas for how to do it.

Also, this would add it to the getConfig route, not the getClientConfig route. I can't think of any reason a non-admin would need to know this

Reply:
I agree with the discussion about this on pre-release. A separate route should be fine.